packages/cardpay-sdk/sdk/scheduled-payment/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { hubRequest, poll } from '../utils/general-utils';
+
+export async function waitUntilSchedulePaymentTransactionMined(
+  hubRootUrl: string,
+  scheduledPaymentId: string,
+  authToken: string
+) {
+  return poll(
+    () => hubRequest(hubRootUrl, `api/scheduled-payments/${scheduledPaymentId}`, authToken, 'GET'),
+    (response: any) => response.data.attributes['creation-block-number'] != null,
```

Comment:
Maybe we can add an error field to the scheduled_payments table and let the background worker update it in case of a revert. The poll can then check for that too.

Reply:
good idea

Reply:
Added